mailer.rst

```diff
@@ -160,6 +160,11 @@ party provider:
     For example, the DSN ``ses+smtp://ABC1234:abc+12/345@default`` should be
     configured as ``ses+smtp://ABC1234:abc%2B12%2F345@default``
 
+.. caution::
+
+    Symfony 4.4 only suppports Amazon SES Signature Version 3, that has been deprecated.
+    You need to use symfony/amazon-mailer 5.1 or newer.
```

Comment:
```suggestion
    You need to use symfony/amazon-mailer 5.2 or newer.
```

as `5.1` is EOM

Reply:
It is, but problem was indeed fixed in 5.1
If we stick to supported version, we'll need to change such value every time a version is out of support

Reply:
I thought about this too. This PR should have been made last year. So I believe 5.1 is correct.

Unless it causes issues while merging.

Reply:
Fair enough